### PR TITLE
09-BUNSPACE2024-Create-Basic-Layout-For-Discord

### PR DESCRIPTION
**Pull request by jmorales8**
Completes: https://github.com/jmorales8/Bunnys-space/issues/9

### CODE REVIEW

**Comment by GrpFruit** [comment]
I think there is too much space in between discord cards, maybe try adding a 3rd one or making the cards bigger

File: ui/src/pages/Discord/Servers/Bunzone.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { ServerTemplate } from "./ServerTemplate/ServerTemplate";
+
+export function Bunzone() {
+  return(
+    <>
+      <ServerTemplate title="The Bunzone" description="" img="bunnyL.jpg" joinLink="discord/bruh" />
+    </>
```
Comment (GrpFruit) [inline ui/src/pages/Discord/Servers/Bunzone.tsx:7]: Get rid of `<>` and `</>` and write line 6 as:
`<ServerTemplate
      title="The Bunzone"
      description=""
      img="bunnyL.jpg"
      joinLink="discord/bruh"
    />`

Comment (GrpFruit) [inline ui/src/pages/Discord/Servers/Bunzone.tsx:7]: Get rid of <> and </> and write line 6 as:
```suggestion
    <ServerTemplate
      title="The Bunzone"
      description=""
      img="bunnyL.jpg"
      joinLink="discord/bruh"
    />
```

File: ui/src/pages/Discord/Servers/NightRunners.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { ServerTemplate } from "./ServerTemplate/ServerTemplate";
+const text = "Anya Forger: The adopted daughter of Loid Forger and Yor Forger, who doesn't know she isn't Loid's biological daughter. She was used as an experiment by the Ostanian government that gave her the ability to read minds. She ended up"
+
+export function NightRunners() {
+  return (
+    <>
+      <ServerTemplate title="Night Runners" description={text} img="bunnyL.jpg" joinLink="discord/deeznuts" />
+    </>
```
Comment (GrpFruit) [inline ui/src/pages/Discord/Servers/NightRunners.tsx:9]: Similar to the one for Bunzone

**Comment by jmorales8** [comment]
> Collaborator

I dont know what the third one could be. I could change it to be different so that it's by px and not %. Or I could just give them less space

**Comment by GrpFruit** [comment]
LGTM +1

**Pull request merged**